.travis.yml

```diff
@@ -1,11 +1,19 @@
-language: python
-install:
- - git clone https://github.com/w3c/respec.git
- - git clone https://github.com/dontcallmedom/webidl-checker.git
- - git clone https://github.com/dontcallmedom/widlproc
- - cd widlproc && make obj/widlproc && cd ..
- - pip install html5lib lxml
+language: node_js
+
+sudo: false
+
+branches:
+  only:
+    - TR
+
+env:
+  global:
+    - URL="https://rawgit.com/w3c/mediacapture-depth/TR/ECHIDNA"
```

Comment:
Not sure if that explains the bug we've encountered, but can you try with http://htmlpreview.github.io/?https://github.com/w3c/mediacapture-depth/TR/ECHIDNA instead? (the publication token will need to be updated in this case I think)